specs/046-tdd-verify-red/contracts/verify-red.md

```
@@ -0,0 +1,79 @@
+- classification: assertionFailure
+- criterion: FR-007
+- test: test/plugins/tdd/fixtures/b_003_test.dart
+- command: `dart test test/plugins/tdd/fixtures/b_003_test.dart --plain-name "..."`
+- exit: 1
+- at: 2026-08-30T12:00:00.000Z
+- output:
+```text
+Expected: ...
+  Actual: ...
+```
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)